README.md

```diff
@@ -1,122 +1,209 @@
 
-### Trace continuation and propagation
+### Complete Example
 
-Instana Go Collector ensures that application trace context will continued and propagated beyond the service boundaries using various
-methods, depending on the technology being used. Alongside with Instana-specific HTTP headers or message attributes, a number of open
-standards are supported, such as W3C Trace Context and OpenTelemetry.
+[Basic Usage](./example/basic_usage/main.go)
+```go
+package main
 
-#### W3C Trace Context & OpenTelemetry
+import (
+  "log"
+  "net/http"
 
-The instrumentation wrappers provided with Go Collector automatically inject and extract trace context provided via W3C Trace Context HTTP
-headers.
+  instana "github.com/instana/go-sensor"
+)
 
-### Continuous profiling
+func main() {
+  col := instana.InitCollector(&instana.Options{
+    Service:           "Basic Usage",
+    EnableAutoProfile: true,
+  })
 
-[Instana AutoProfile™][docs.autoprofile] generates and reports process profiles to Instana. Unlike development-time and on-demand profilers,
-where a user must manually initiate profiling, AutoProfile™ automatically schedules and continuously performs profiling appropriate for
-critical production environments.
+  http.HandleFunc("/endpoint", instana.TracingHandlerFunc(col, "/endpoint", func(w http.ResponseWriter, r *http.Request) {
+    w.WriteHeader(http.StatusOK)
+  }))
+
+  log.Fatal(http.ListenAndServe(":7070", nil))
+}
+```
 
-Please refer to the [Instana Go Collector docs](https://www.ibm.com/docs/en/obi/current?topic=go-collector-common-operations#instana-autoprofile%E2%84%A2) to learn how to activate and
-use continuous profiling for your applications and services.
+### Wrapping up
 
-### Sending custom events
+Let's quickly summarize what we have seen so far:
 
-The Go Collector, be it instantiated explicitly or implicitly through the tracer, provides a simple wrapper API to send events to Instana as described in [its documentation](https://www.ibm.com/docs/en/obi/current?topic=integrations-sdks-apis).
+1. We learned how to install, import and initialize the Instana Go Tracer.
+1. Once the tracer is initialized application metrics are collected out of the box.
+1. Application profiling can be enabled via the `EnableAutoProfile` option.
+1. Tracing incoming HTTP requests by wrapping the Go standard library `http.Handler` with `instana.TracingHandlerFunc`.
 
-To learn more, see the [Events API](./EventAPI.md) document in this repository.
+With this knowledge it's already possible to make your Go application traceable by our SDK.
+But there is much more you can do to enhance tracing for your application.
 
-## Examples
+In total, the Go tracer offers means to trace the following standard Go library features:
 
-Following examples are included in the `example` folder:
+1. HTTP incoming requests
+1. HTTP outgoing requests
+1. SQL drivers
```

Comment:
In my opinion, this kind of make me feel that the features are limited to these three. I do see that in the next line that the support for additional packages are mentioned. I am just expressing my thoughts. You don't have to change it, if you have a different view.

Reply:
@sanojsubran I rephrased it to "The basic functionality covers tracing for the following standard Go features:"
Any thoughts on this?

Reply:
Yes, This looks better to me.